dev-python/nuitka/nuitka-0.6.14.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS="rdepend"
+
+PYTHON_COMPAT=( python3_{7,8} )
```

Comment:
no 3.9? (we're switching next month)

Reply:
That's a good addition! Thanks for reminding me about it. I am already using Python 3.9 ...